doc/intl/fr/README.md

```diff
@@ -43,20 +45,25 @@ Principales ressources de développement :
 
 * [GitHub Discussion](https://github.com/dogecoin/dogecoin/discussions) est utilisé pour
   discuter des caractéristiques, planifiées et non planifiées, liées à la fois au développement du logiciel Dogecoin Core, les protocoles sous-jacents et l'actif DOGE.
-  
+
 * [Dogecoindev subreddit](https://www.reddit.com/r/dogecoindev/)
 
 ### Stratégie de version
+
 Les numéros de version suivent la sémantique ```major.minor.patch```.
 
 ### Branches
-Il y a 3 types de branches dans ce référentiel :
 
-- **master:** Stable, contient la dernière version de la dernière *major.minor* version.
-- **maintenance:** Stable, contient la dernière version des versions précédentes, qui sont toujours en maintenance active. Format : ```<version>-maint```
-- **development:** Instable, contient du nouveau code pour les versions prévues. Format : ```<version>-dev```
+Il y a 4 types de branches dans ce référentiel :
+
+- **master :** Instable, contient le code le plus récent en développement.
+- **maintenance :** Stable, contient la dernière version des versions précédentes, qui sont toujours en maintenance active. Format : ```<version>-maint```
+- **development :** Instable, contient du nouveau code pour les versions prévues. Format : ```<version>-dev``` 
+- **archive :** Stable, branches immuables pour les anciennes versions qui ne changent plus car elles ne sont plus maintenues.
```

Comment:
Saying "immuable" and "qui ne changent plus" is a pleonasm. I would rephrase this as follow:

```suggestion
- **archive :** Stable, branches immuables d'anciennes versions qui ne sont plus maintenues.
```

Reply:
Immuables is referring to branches and qui ne changent plus is referring to les anciennes versions, so each are referring to a specific and distinctive word. But the change you're suggesting does sound better.